app/src/main/java/com/portfolio/tasky/BaseActivity.kt

```diff
@@ -0,0 +1,50 @@
+package com.portfolio.tasky
+
+import android.content.Intent
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+
+open class BaseActivity : AppCompatActivity() {
+    private var mActivity: BaseActivity? = null
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        mActivity = this
+    }
+
+    override fun onAttachedToWindow() {
+        super.onAttachedToWindow()
+        // Not implemented yet
+    }
+
+    override fun onStart() {
+        super.onStart()
+        // Not implemented yet
+    }
+
+    override fun onPause() {
+        super.onPause()
+        // Not implemented yet
+    }
+
+    override fun onStop() {
+        super.onStop()
+        // Not implemented yet
+    }
+
+    override fun onResume() {
+        super.onResume()
+        // Not implemented yet
+    }
+
+    override fun onDestroy() {
+        super.onDestroy()
+        // Not implemented yet
+    }
```

Comment:
No need to add this on this PR if they aren't being used right now

Reply:
Done